conda-forge.yml

```diff
@@ -1,10 +1,9 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
+  linux_ppc64le: linux_ppc64le  # tests fail when emulating power from linux_64
   osx_arm64: osx_64
 conda_forge_output_validation: true
 provider:
-  linux_aarch64: default
+  linux_aarch64: travis
```

Comment:
There is no ARM support on Travis. We are stuck emulating them on Azure. Yes it does take a while unfortunately.

Please see PR ( https://github.com/conda-forge/conda-forge.github.io/pull/1555 )